internal/auth/oidc/repository_auth_method_update.go

```diff
@@ -208,14 +212,15 @@ func (r *Repository) UpdateAuthMethod(ctx context.Context, am *AuthMethod, versi
 				// method's version.
 				updatedAm = am.Clone()
 				updatedAm.Version = uint32(version) + 1
-				rowsUpdated, err = w.Update(ctx, updatedAm, []string{"Version"}, nil, db.NewOplogMsg(&authMethodOplogMsg), db.WithVersion(&version))
+				rowsUpdated, err = w.Update(ctx, updatedAm, []string{"Version", "DisableDiscoveredConfigValidation"}, nil, db.NewOplogMsg(&authMethodOplogMsg), db.WithVersion(&version))
```

Comment:
Could you have a test which:
1. Updates with force
2. Verifies a lookup on the authmethod has DisableDiscoveryConfigValidation set
3. Updating again without setting force
4. a lookup on the authmethod no longer has that value set.

Reply:
done and ty!  While adding the test I found a few other issues.  The biggest one: proto.Clone(...) doesn't appear to properly clone booleans.  See the new commits.